homeassistant/components/glances/sensor.py

```diff
@@ -223,6 +223,20 @@ class GlancesSensorEntityDescription(
         icon="mdi:docker",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ("raid", "status"): GlancesSensorEntityDescription(
+        key="status",
+        type="raid",
+        name_suffix="Raid status",
+        icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
Similar, status is probably a string, so not a measurement. Should this be an enum instead?

Reply:
I jut read the dev docs about state_class needing to be numerical when not None.
From a logical perspective it would also be possible to do measurements of states. That why I was setting state_class to measurement here.
Anyways, changed that - thanks for the hint!

Unfortunately glances api documentation is lacking raid information. Therefore the possible values for status are not clear. Because of that I would prefer not using an enum.

Reply:
I think it would make sense to use a enum for the states we do know.

- The states we don't know, will be reported and thus the enum can be extended at that point
- Translations for the states can be provided.

Reply:
Currently the only known status to me is `active`... Should I create an enum anyways? This would lead to an Exception being thrown when trying to set a missing status, right?

Reply:
For resolving the chage request I need an information.
Should I start creating an enum with just one value and then enhance it when new states are reported by users or should I leave it as is just removing the wrong state class?